pycqed/analysis_v2/base_analysis.py

```diff
@@ -269,9 +269,10 @@ def _raise_warning(self):
         :param warning_textfile_name: string with name of the file without
             extension.
         """
-        if not self.check_plotting_delegation():
+        if self.check_plotting_delegation():
+            # Only execute this function if the AnalysisDaemon is not running.
             # The warning image and text file will be generated twice
-            # when the AnalysisDaemon is active
+            # when the AnalysisDaemon is running.
```

Comment:
I find the current comment a bit confusing. Would the following be an accurate description of the return statement?

The current analysis call delegates plotting to the analysis daemon, and the warning image and text file will be generated by the daemon.

Reply:
I changed it now, thanks.